### PR TITLE
Sfl 93

### PR DESCRIPTION
**Pull request by saketPandhare-Webonise**
Add Ability to show Fav in fav-listing and remove item from fav listing

### CODE REVIEW

File: SUNFLOWR/SUNFLOWR/MoveVC/MoveViewModel.swift [hunk 1]
```diff
@@ -104,27 +119,29 @@ class MoveViewModel {
...
     
-    /// This function is used to get parent id of Fav card
-    /// - Parameter rowNo: row no of fav card
-    /// - Returns: parentID
-    func getParentCardOfFavMove(rowNo: Int) -> String {
-        if (moveFavInfoModel.data.count > rowNo) {
-            return moveFavInfoModel.data[rowNo].id
-        } else {
-            return ""
+    func checkIfCardIsFav(rowNo: Int) -> Bool {
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/MoveVC/MoveViewModel.swift:129]: Add Method documentation here.

File: SUNFLOWR/SUNFLOWR/MoveVC/MoveViewModel.swift [hunk 2]
```diff
@@ -104,27 +119,29 @@ class MoveViewModel {
...
+        if (rowNo < arrayMoveObjectData.count) {
+          return arrayMoveObjectData[rowNo].attributes.favourite
+        }else {
+            return false
         }
     }
     
+    func updateFavStatusInLocalArray(rowNo: Int, isFav: Bool) {
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/MoveVC/MoveViewModel.swift:137]: Add Method documentation here.

Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/MoveVC/MoveViewModel.swift:144]: Done

File: SUNFLOWR/SUNFLOWR/MoveVC/MoveViewModel.swift (hunk 1 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/MoveVC/MoveViewModel.swift:132]: Done

**Pull request merged**